backend/src/app.js

```diff
@@ -197,20 +198,32 @@ router.post('/v1/templates/:name/send/:stmpSelected?', async (ctx) => {
     let subjectTemplatePath = path.join(templatePath, `${ctx.request.body.lang || 'default'}.subject`);
     let mjmlTemplatePath = path.join(templatePath, `${ctx.request.body.lang || 'default'}.mjml`);
     let txtTemplatePath = path.join(templatePath, `${ctx.request.body.lang || 'default'}.txt`);
+    let jsonSchemaPath = path.join(templatePath, 'schema.json');
     const attachmentsPath = path.join(
         config.get('template_path'),
         ctx.params.name,
         'attachments'
     );
 
-    if (ctx.request.body.lang && (!fs.existsSync(subjectTemplatePath) || !fs.existsSync(mjmlTemplatePath) || !fs.existsSync(txtTemplatePath))) {
+    if (
+        ctx.request.body.lang && (
+            !fs.existsSync(subjectTemplatePath) ||
+            !fs.existsSync(mjmlTemplatePath) ||
+            !fs.existsSync(txtTemplatePath)
+        )
+    ) {
         console.warn(`Can't find all "${ctx.request.body.lang}" files for this template, fallbacking to default language`);
         subjectTemplatePath = path.join(templatePath, 'default.subject');
         mjmlTemplatePath = path.join(templatePath, 'default.mjml');
         txtTemplatePath = path.join(templatePath, 'default.txt');
     }
 
-    const templates = [subjectTemplatePath, mjmlTemplatePath, txtTemplatePath];
+    const templates = [
+        subjectTemplatePath,
+        mjmlTemplatePath,
+        txtTemplatePath,
+        jsonSchemaPath
```

Comment:
@Its-Alex **suggestion:** what do you think about make `jsonSchemaPath` file optional? If `jsonSchemaPath` file missing, then schema validation is skipped with a `console.info` messages 🤔

Reply:
@Its-Alex Bad idea, this file is mandatory for frontend formulaire rendering.